src/main/resources/templates/signup2.html

```diff
@@ -18,6 +18,9 @@ <h3 class="dashhead loginpanel" th:text="#{signup.header}">Registration</h3>
 							<p> <b>Come join NCL. Let's set up your account.</b></p>
 						</div>
                         <div class="panel-body">
+							<div th:if="${message}">
+								<p><i class="fa fa-exclamation-circle btn-fa-red"></i> <span class="account-change-fail" th:text="${message}"></span></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)